modules/ruxruntime/src/lib.rs

```diff
@@ -199,7 +199,10 @@ pub extern "C" fn rust_main(cpu_id: usize, dtb: usize) -> ! {
     ruxhal::platform_init();
 
     #[cfg(feature = "multitask")]
-    ruxtask::init_scheduler();
+    {
+        ruxtask::init_scheduler();
+        ruxfutex::init_futex();
```

Comment:
We do not need futex-related structure if `musl` feature is not enabled. Please put `init_futex` under `musl` feature.

Reply:
Done.